src/stage.js

```diff
@@ -86,12 +86,12 @@ class TouchRecord {
             // Click on object to reduce
             let selectedNode = this.topNode;
 
-            if (this.targetNode) {
+            /*if (this.targetNode) {
```

Comment:
Heads up, you can just go ahead and delete the code - we can always restore it from Git if needed (just use a descriptive commit message so it's easy to find)